Gemfile

```diff
@@ -6,6 +6,7 @@ group :test do
   gem "simplecov"
   gem "codeclimate-test-reporter"
   gem "byebug", platform: :mri
+  gem "dry-container"
```

Comment:
Good point @timriley, indeed, it makes more sense to have it here. As you stated, dry-container isn't a runtime dependency.